.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2.1
```

Comment:
Out of interest, why circleci and not github actions?

Reply:
just have never used it before, wanted to try :)